spice/cfg/tests-variants.cfg

```diff
@@ -165,6 +165,22 @@ variants:
                             ticket_send = 12456
                             include join.cfg
 
+                - 527185:
+                    # only for qemu-kvm-rhev testing
```

Comment:
So, make this knowledge available to all )
# As for Feb 2017  RHEL 7 qemu doesn't listen on IPv6.
Great comment!)